Feature/import score ini

Ashiro12138 opened the pull request:
# Description
- Added a button in settings to import scores to database
# Screenshots
![image](https://github.com/user-attachments/assets/85e26365-0327-4d11-becd-36c8656cceac)

DragonRatTiger reviewed (changes requested):
Looks great so far, I'd like to request a few changes :

- The function rename should be reverted. `ImportScoreInisToSavesDb3()` itself is not a typo, since it refers to `*.score.ini` files.
- In addition to making sure these keys exist in every `lang.json` file, I recommend renaming them accordingly to fit the existing naming convention :
  - `"SETTINGS_SYSTEM_IMPORT_SCORE_INIT"` -> `"SETTINGS_SYSTEM_IMPORTSCOREINI"`
  - `"SETTINGS_SYSTEM_IMPORT_SCORE_INIT_DESC"` -> `"SETTINGS_SYSTEM_IMPORTSCOREINI_DESC"`
- The button existing is great, but there doesn't seem to be any visual indicator for import progress, or anything that stops the user from continuing to browse the Settings menu in the process. If you aren't sure how to get this to work, you can leave it as-is for this PR.

Ashiro12138 commented:
I will address these comments tonight

0auBSQ commented:
Additionally, as it is a pretty sensitive feature (directly interacts with the database save) there should probably be a warning text either in red or yellow (using color tags like <c.#ff0000> ... </c>)
The clear statuses/badges not being transfered should also be mentionned

DragonRatTiger commented:
Right, there are some things that won't make it into the new database.
In particular, these won't be transferred :
- Clear statuses
- Gameplay modifiers
- # of hits (Good, Ok, Bad, Drumroll, Adlib, Bomb, etc.)
- Dan exam scores

Ashiro12138 commented:
I've addressed all comments except for the loading bar one. I will need to look into it more to see if it's possible.
Also still very happy to take change suggestions to the description, I choose a very generic orange warning color.
